apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -84,6 +84,8 @@ function processPost(post, page, api, context) {
   const { locale } = context;
   return {
     ...post,
+    // TODO REMOVE T
+    content: post.content ?? null,
```

Comment:
What is happening here?

Reply:
My payload cms set up was messed up.
I will remove this.